test/index.js

```diff
@@ -116,7 +115,7 @@ test('Grunt-localizr', function (t) {
         });
     });
     t.on('end', function() {
-        process.chdir(original);
+        process.chdir(path.join(__dirname, '..'));
```

Comment:
Old way: from wherever the cwd was to cwd + test/fixtures, then back to wherever the cwd was.

New way: from wherever the cwd was to the text/fixtures relative to the file, then to two levels up.